### ISSUE TITLE
rbac version in deployment yamls should be v1 (not v1beta1)

### ISSUE THREAD

**Issue by soamvasani** [fission/fission#476]
```
Error: release rude-crocodile failed: the API version in the data (rbac.authorization.k8s.io/v1beta1) does not match the expected API version (rbac.authorization.k8s.io/v1)
```
@sarun87 found this bug and is working on it; thanks!

**Comment by soamvasani** [comment]
I don't understand why this doesn't fail on our CI cluster (1.8.5), though.

**Comment by sarun87** [comment]
PR for this - https://github.com/fission/fission/pull/477

**Comment by life1347** [comment]
Fixed by #1124
